### ISSUE TITLE
Improving the user experience

### ISSUE THREAD

**Issue by Cizall** [Mytherin/Tibialyzer#145]
I would like to suggest a new design for Tibialyzer. While playing around with the design and information architecture I've found that the application is sometimes inconsistent. For instance there are settings for "popups" in the "Settings" tab, however more settings for popups are available under the "popup" tab. This kind of inconsistency can be found on more places in the application and could be improved greatly. 
Furthermore I would like to put in a request to redesign the entire application in all of the above mentioned categories, design, information architecture and user experience. So far I've only played around a little with the colors of the application and made some adjustments in terms of having equally large labels and so on throughout. I also tried out a new logo but I don't know if anyone agrees with that change!

I believe this application has a lot to offer and is very well made! Although I also believe that the user experience hasn't been the highest priority when creating it, which is why a new user has a bit of trouble actually understanding how all of it works.

I hope to be able to contribute with this to the project and that everyone is willing to help out with feedback and requests on how they would like to see this part of the project go forward.

![image](https://cloud.githubusercontent.com/assets/13249858/17082637/301fef74-5186-11e6-8d9e-e1cb688959dc.png)

**Comment by Mytherin** [comment]
I like the look of your new design! Indeed, as they say, developers can't make good interfaces. As such, Tibialyzer's interface is not very well designed. If you think you can improve on the interface, I think that would be a very welcome change for many people. So please, if you want to revamp the interface, be my guest!

Not a big fan of the new logo, although the logo could definitely be changed. But this logo looks a bit too low-quality, and I'm not a big fan of the colors (white + green).

**Comment by Cizall** [comment]
Great! 

I'll see what I can do!
Yea I'm starting to hate the logo as well.. 

Do you think changing up where the different settings are found will be a problem? I  mean like moving them to a different "tab"?

I was also thinking about using up more screen-realty while the user is "setting" up everything, cause they probably wont be in a need to see whats going on behind the windows while in there.

I'll make some "mockups" to see if there is one style we can agree upon! @Mytherin

**Comment by Mytherin** [comment]
You can move settings around all you want. The groupings now are mostly the result of constantly just adding settings where-ever there is room. As a result, many groupings are not logical at all. I think you would need to move them around if you want to have good results.

Increasing the window size by a little is fine, but you have to keep in mind that some people do not have big screens. So try to keep the window size down, to at most something like 800x600.

**Comment by Cizall** [comment]
Great :) Yea I think so too..

Yea I'm worrying about that as well.. Is there a easy way to make the windows form a bit more "responsive"? Is the change to WPF too big a step to make this far into development? @Mytherin

**Comment by Mytherin** [comment]
If you want to use WPF then go ahead; the only reason I used Windows Forms was because it was easiest for me since I have used that a lot before. It might take too much effort to make the change, though.

**Comment by Cizall** [comment]
Yeah, I'm guessing changing it will be a paint in the butt.. And incorporating XAML into it now would be as well I guess. Not very familiar with it myself either to be honest!
I would however love to have some kind of standard controls to be used in the application so that changing visuals is easier in the future. Have any tips on how I could accomplish that?

What do you think about something like this for the logo?

![tibialyzer sidestaplar2](https://cloud.githubusercontent.com/assets/13249858/17156084/34337622-5388-11e6-8d4c-b31283d53c48.png)
![tibialyzer sidestaplar4](https://cloud.githubusercontent.com/assets/13249858/17156087/38fe79a4-5388-11e6-8496-21426fe7825c.png)
![tibialyzer sidestaplar5](https://cloud.githubusercontent.com/assets/13249858/17156090/3b258ede-5388-11e6-8963-68bdf1d0c32a.png)

**Comment by Mytherin** [comment]
I would suggest making your own class for each control. Then you can inherit from the base control you want to make (e.g. if you want to make a slider you inherit from the Slider control). Then instead of adding Sliders to your forms you add CustomSliders. You can then add styling information/etc. to the CustomSlider class. 

I think that logo looks very nice, I like the first one especially.

**Comment by Mytherin** [comment]
Have you made any progress on the UI revamp? I want to do some work on it myself, but I don't want to mess with any work you have been doing.
